c11threads.h

```diff
@@ -158,10 +162,27 @@ static inline int mtx_trylock(mtx_t *mtx)
 static inline int mtx_timedlock(mtx_t *mtx, const struct timespec *ts)
 {
 	int res;
+#ifdef __APPLE__
+	struct timeval now;
+	struct timespec sleeptime;
+
+	sleeptime.tv_sec = 0;
+	sleeptime.tv_nsec = 5000000; // 5 ms
```

Comment:
For now at least I'll make it a constant, so that it's not just a random magic number.